### PR TITLE
Infoblox new solution - SOC Insights

### PR DESCRIPTION
**Pull request by sschuur**
Required items, please complete
   
   Change(s):
   - New solution - Infoblox SOC Insights integration
   - 3 data connectors, 6 parsers, 2 analytic rules, 3 playbooks, 1 workbook

2 different ways to import our new SOC Insight data - either with the REST API using the playbook provided in the solution, or by using the Infoblox CDC to send CEF to Sentinel with the Microsoft forwarding agent (AMA or OMS/MMA). Insights get added as Incidents in sentinel where playbooks can be run on them to get further details. Details are used to build workbook and can be used anywhere else in sentinel as desired of course.

- added sample data in [Sample Data](https://github.com/sschuur/Azure-Sentinel/tree/master/Sample%20Data)
- added entry to [WorkbookMetadata.json](https://github.com/sschuur/Azure-Sentinel/blob/master/Workbooks/WorkbooksMetadata.json)

   Reason for Change(s):
   - New solution

   Testing Completed:
   - Tested in personal environment

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @sschuur, Thanks for raising this PR. This PR will be investigated and we will provide you an update about the same before 11 March, 2024

**Comment by sschuur** [comment]
Hi @v-prasadboke , thank you!

I can't seem to pass the KQL tests. Something is wrong with my [CustomTables](https://github.com/sschuur/Azure-Sentinel/blob/InfobloxSOCInsights-New-Solution/.script/tests/KqlvalidationsTests/CustomTables) & how i am referring to them in the [InfobloxSOCInsightsDataConnector_API.json](https://github.com/sschuur/Azure-Sentinel/blob/InfobloxSOCInsights-New-Solution/Solutions/Infoblox%20SOC%20Insights/Data%20Connectors/InfobloxSOCInsightsDataConnector_API.json). Getting a little stuck.

**Comment by v-prasadboke** [comment]
> Hi @v-prasadboke , thank you!
> 
> I can't seem to pass the KQL tests. Something is wrong with my [CustomTables](https://github.com/sschuur/Azure-Sentinel/blob/InfobloxSOCInsights-New-Solution/.script/tests/KqlvalidationsTests/CustomTables) & how i am referring to them in the [InfobloxSOCInsightsDataConnector_API.json](https://github.com/sschuur/Azure-Sentinel/blob/InfobloxSOCInsights-New-Solution/Solutions/Infoblox%20SOC%20Insights/Data%20Connectors/InfobloxSOCInsightsDataConnector_API.json). Getting a little stuck.

Hello @sschuur, Looking into the issue

**Comment by v-prasadboke** [comment]
Hello @sschuur, Sorry for the inconvenience. 
But due to lack of unavailability we were not able to touch this PR today. We will come back to you by 14 March, 2024

**Comment by v-prasadboke** [comment]
Hello @sschuur, We have skip validated the template. Continuing to test the PR. 
Thanks.

**Comment by v-prasadboke** [comment]
Hello @sschuur =, We are facing an issue while custom deploying the maintemplate.
Content hub is throwing an react error.

Can you try to deploy the maintemplate from your end and check if its working.
Thanks.

File: Solutions/Infoblox SOC Insights/Playbooks/Infoblox-SOC-Import-Indicators-TI/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+            "1. Entra ID Application Secret",
+            "2. Client ID",
+            "3. Tenant ID"
+        ], 
+        "postDeployment":  [
+            "1. Grant playbook's Managed Identity **Microsoft Sentinel Responder** or greater to Resource Group.",
+            "2. Authorize connections."
+        ],
+        "prerequisitesDeployTemplateFile":  "",
+        "lastUpdateTime":  "",
```
Comment (v-prasadboke) [inline Solutions/Infoblox SOC Insights/Playbooks/Infoblox-SOC-Import-Indicators-TI/azuredeploy.json:17]: Please update last update time

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**